config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -426,3 +426,97 @@ presubmits:
       testgrid-num-failures-to-alert: '10'
       testgrid-alert-stale-results-hours: '24'
       testgrid-create-test-group: 'true'
+
+periodics:
+- name: periodic-kubernetes-e2e-kind-flakes
+  interval: 2h
+  labels:
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decoration_config:
+    timeout: 60m
+  path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/krte:v20200123-dd36597-master
+      command:
+      - wrapper.sh
+      - bash
+      - -c
+      - gsutil cp -P gs://bentheelder-kind-ci-builds/latest/kind-linux-amd64 "${PATH%%:*}/kind" && gsutil cat gs://bentheelder-kind-ci-builds/latest/e2e-k8s.sh | sh
+      env:
+      - name: FOCUS
+        value: \[Flaky\]
```

Comment:
Please verify the regex are correct?

Reply:
sure, but I'm not clear that the idea is even useful